messages/recordcounts.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commandDescription": "Use this command to get an approximate count of the records in standard or custom objects in your org. These record counts are the same as the counts listed in the Storage Usage page in Setup. The record counts are approximate because they're calculated asynchronously and your org’s storage usage isn’t updated immediately.",
```

Comment:
One last edit.  And I apologize for not making this clear originally...

Please put back the original short sentence (all lower case and not punctuation) for the short description that's displayed first thing when a user runs the command with --help.  Then add a "\n" and add the rest of the description that starts with "Use this command...".  

So **commandDescription** will look something like this:

"commandDescription": "display record counts for the specified standard and custom objects\nUse this command to get an approximate count of the records in standard or custom objects in your org. These record counts are the same as the counts listed in the Storage Usage page in Setup. The record counts are approximate because they're calculated asynchronously and your org’s storage usage isn’t updated immediately."

See the **description** object for the [auth:jwt:grant](https://github.com/salesforcecli/plugin-auth/blob/main/messages/jwt.grant.json) command for an example. 

Thanks!

Reply:
@WillieRuemmele do you want to take the text over from my last commit on the original PR? There the text already contains this split.